tests/settings/e2e/steps/stakepool-settings.js

```diff
@@ -0,0 +1,84 @@
+import { When, Then,} from 'cucumber';
+
+const CHANGES_SAVED_SUCCESS_MESSAGE = '//*[@class="StakePoolsSettings_savingResultLabel"]';
+const CUSTOM_SERVER_DROPDOWN_OPTION = '//*[@class="SimpleInput_customValueWrapper"]//*[@value="Custom server"]';
+const DAEDALUS_TOP_BAR_LOGO = '//*[@class="TopBar_topBar TopBar_withoutWallet"]';
+const IS_ERROR_MESSAGE_ACTIVATED = '//*[@class="SimpleInput_input SimpleInput_errored"]';
+const NONE_FETCH_DATA_DIRECTLY_OPTION = '//*[@class="SimpleInput_customValueWrapper"]//*[@value="None - fetch the data directly"]';
+const NOT_A_VALID_SMASH_SERVER_ERROR_MESSAGE_ACTIVATED = '//*[@class="InlineEditingInput_errorMessage" and text()="This URL is not a valid SMASH server"]';
+const OFF_CHAIN_METADATA_SERVER_SMASH_LABEL = '//*[contains(text(), "Off-chain metadata server (SMASH)")]';
+const SMASH_SERVER_URL_INPUT_BOX = '//*[@label="SMASH server URL"]';
+const STAKE_POOL_CUSTOM_SERVER_INPUT_BOX_SUBMIT_BUTTON = '//*[@class="InlineEditingInput_button InlineEditingInput_okButton SimpleButton_root ButtonOverrides_root"]';
+const STAKE_POOL_CUSTOM_SERVER_INPUT_BOX_X_BUTTON = '//*[@class="InlineEditingInput_button InlineEditingInput_cancelButton SimpleButton_root ButtonOverrides_root"]';
+const STAKE_POOL_SERVER_DROPDOWN = '//*[@class="SimpleFormField_inputWrapper"]';
+const STAKE_POOL_SERVER_DROPDOWN_CUSTOM_OPTION = '//*[@class="ScrollbarsCustom-Content"]//span[text()="Custom server"]';
+const STAKE_POOLS_SUBMENU_SETTINGS = '//*[@class="SettingsMenu_component"]//button[text()="Stake pools"]';
+
+When(/^none fetch the data directly is the default option$/, function() {
+  return this.waitAndClick(NONE_FETCH_DATA_DIRECTLY_OPTION);
+});
+
+When(/^custom server is the default option$/, function() {
+  return this.waitAndClick(CUSTOM_SERVER_DROPDOWN_OPTION);
+});
+
+When(/^I click on Daedalus logo to change focus$/, function() {
+  // This step was necessary as when the error message for this box is displayed the submit button could not receive the click and i got a "Element is not clickable at point error"
+  return this.waitAndClick(DAEDALUS_TOP_BAR_LOGO);
+});
+
+Then(/^"([^"]*)" is visible on stake-pool screen above stake-pool list and is clickable$/, function(serverUrl) {
+  return this.waitAndClick('//*[@class="StakePools_smashSettings"]//span[text()="Moderated by '+ serverUrl + '"]');
```

Comment:
If you want to write modern Javascript you could use Template_literals this instead string concatenation https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals 😉

I would look like this: 
<img width="756" alt="image" src="https://user-images.githubusercontent.com/4303099/125747487-4a4e7ed3-8850-4af7-931c-66340be4f2dd.png">

Reply:
That was my first approach but unfortunately it ends up not recognising the ${serverUrl} variable because it sees it as a normal string as its inside the "Moderated by....." quotation marks. Can you please advise a way that I can get around this ? I used the string concatination above as an alternative solution

<img width="981" alt="Screenshot 2021-07-16 at 16 50 28" src="https://user-images.githubusercontent.com/56027085/125975253-83bba93f-25f9-4c6c-b491-1143de5093d3.png">